blinkist-airbrake-scrubber.gemspec

```diff
@@ -14,7 +14,7 @@ Gem::Specification.new do |gem|
   gem.license       = "MIT"
 
   # Airbrake
-  gem.add_dependency "airbrake", "~> 9"
+  gem.add_dependency "airbrake", ">= 9", "< 11"
```

Comment:
`~> 10` ??

Reply:
I didn't want to "force" to upgrade to 10. With my version you can use this gem with 9 and 10 as you like.
Can also force to 10 if you think that's viable across the backend systems.